### PR TITLE
Feat/tabs

### PR DESCRIPTION
**Pull request by chrisrhoden**
the filepicker and edit sheet both make use of this tabs ui component. breaking it out into a PR for easier review

### CODE REVIEW

**Comment by chrisrhoden** [comment]
@kookster assigned to @farski but if you have a chance to poke at it as well let me know

File: src/app/ui/tabs/tabs.styl [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+        padding-right: 23px;
+      }
+      &::after {
+        height: 4px;
+        margin-top: -2px;
+        border-width: 2px;
+      }
+    }
+
+    li.active-remove, li.active-add {
```
Comment (farski) [inline src/app/ui/tabs/tabs.styl]: two lines

File: src/app/ui/tabs/tabs.styl [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+    > * {
+      transition: opacity 0.1s linear;
+      position: absolute;
+      top: 0; left: 0;
+      &.ng-enter, &.ng-leave-active {
+        opacity: 0;
+      }
+
+      &.ng-leave, &.ng-enter-active {
```
Comment (farski) [inline src/app/ui/tabs/tabs.styl]: two lines

File: src/app/ui/tabs/tabs.styl [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  > div {
+    flex: 1.8 0 0%;
+    height: 100%;
+    position: relative;
+
+    > * {
+      transition: opacity 0.1s linear;
+      position: absolute;
+      top: 0; left: 0;
+      &.ng-enter, &.ng-leave-active {
```
Comment (farski) [inline src/app/ui/tabs/tabs.styl]: two lines

File: src/app/ui/tabs/tabs.styl [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+  > ul {
+    display: flex;
+    background-color: $tab-bed-color;
+    overflow-x: auto;
+    border-radius: $tab-border-radius $tab-border-radius 0 0;
+
+    li {
+      flex: 1 0 auto;
+      margin-right: 10px;
+      &:last-child {
```
Comment (farski) [inline src/app/ui/tabs/tabs.styl]: line break

File: src/app/ui/tabs/tabs.styl [hunk 5]
```diff
@@ -0,0 +1,182 @@
...
+      left: 0;
+      margin-top: -11px;
+      border-width: 11px;
+    }
+
+    li:not(.active):hover {
+      a {
+        padding-right: 23px;
+      }
+      &::after {
```
Comment (farski) [inline src/app/ui/tabs/tabs.styl]: line break

**Pull request merged**